README.md

```diff
@@ -4,20 +4,52 @@
 
 Empathic screens for the masses.
 
-Corsica is a server for coordinating screens via web browsers. Corsica recieves content from an API or a plugin, processes it with one or more plugins, and sends output to screens.
+## Looking for Mozilla's Corsica deployment?
 
-Corsica can trivially be used to control ambient displays in an office via an API. Chat bots can yield control of nearby screens to nearby desks, temporarily or indefinitely, for posting graphs, GIFs, or public announcements. Plugins can provide a regular rotation between a list of URLs for ambient dashboard display, like application metrics, promotional pages, or event calendars for the proximal space.
+Suggest content, plugin changes, and administrative changes here: https://github.com/mozilla/moz-corsica
 
-Since Corsica can push content to any modern webbrowser, clients can be monitors, tablets, or phones. Wall mounted tablets are significantly cheaper than TVs with dedicated computers. Phones can be used as transient clients to set up remote viewports when you're away from the relavent screen.
+## What is Corsica? How should I use it?
 
-Plugins make it easy to extend and add new functionality. URLs can be analyzed and reformatted to make presentation on a screen easier -- transform imgur or webcomic urls so that only the image displays, ignoring the cruft of the screen. Transform videos so that they display full screen and autoplay. Create shorthand syntaxes that unpack to whole websites:
+Corsica is a server for coordinating screens via web browsers. Content comes from an API or a plugin, is processed by one or more plugins, and sent as output to connected screens.
 
+Chat bots, browser plugins, or website widgets can use the API to control nearby screens. This is useful for Posting graphs, GIFs, slides, or public announcements. Plugins provide a playlist-like rotation from a list of URLs for ambient dashboards of application metrics, promotional pages, event calendars, etc.
+
+Since Corsica can push content to any modern webbrowser the screens can be monitors, tablets, phones, smart TVs, cars, etc.
+
+## How do I extend Corsica?
+
+Corsica aspires to a tiny core with most features implemented via plugins. Changes to the core can be submitted on this repo as PRs.
+
+Plugins are the easiest way to customize your installation and add new functionality. They are usually developed in an independent package and installed via NPM, though they can also be included directly as files.
```

Comment:
You should mention here that plugins should have the keyword "corsica" on npm, and give a link to [this search](https://www.npmjs.com/browse/keyword/corsica). That helps with the "how can I find plugins" problem.